extract-policies-from-repos.yaml

```diff
@@ -0,0 +1,40 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: quay.io/s_block/alpine-git-jq
```

Comment:
move to MOJ?

Reply:
Cool, has python as well now so will rename as well

Reply:
Moved to moj